createdisk-library.sh

```diff
@@ -383,3 +383,18 @@ function download_podman() {
       mv podman-remote/windows/podman-${version}/usr/bin/podman.exe  podman-remote/windows
     fi
 }
+
+# As of now sparsify helper is very specific to OCP/OKD kind of bundle where we get the
+# partition for the root label and then mount it with guestfish to cleanup /boot. With
+# microshift vm we are using lvm and guestfish error out during mount
+# mount /dev/sda3 /
+# libguestfs: error: mount: mount exited with status 32: mount: /sysroot: unknown filesystem type 'LVM2_member'
+# There might be other way for guestfish to mount lvm but as of now using a seperate helper is easy.
+function create_sparse_qemu_image {
```

Comment:
Maybe name this `sparsify_lvm` if it's the main reason for it. I'd move it to `sparsify` though, and use one codepath or the other depending on `BUNDLE_TYPE`. This way all the code is in a single place.